server/src/test/java/io/crate/integrationtests/MetadataTrackerITest.java

```diff
@@ -297,6 +299,35 @@ public void test_dropped_table_is_replicated() throws Exception {
         });
     }
 
+    @Test
+    public void test_drop_publication_stops_the_replication_on_subscriber() throws Exception {
+        executeOnPublisher("CREATE TABLE t1 (id INT)");
+        executeOnPublisher("INSERT INTO t1 (id) VALUES (1), (2)");
+        createPublication("pub1", true, List.of("t1"));
+        createSubscription("sub1", "pub1");
+
+        assertBusy(() -> assertThat(isTrackerActive(), is(true)));
+        var response = executeOnPublisher("SELECT * FROM pg_publication WHERE pubname = 'pub1'");
+        assertThat(response.rowCount(), CoreMatchers.is(1L));
+
+        executeOnPublisher("DROP PUBLICATION pub1");
+        assertBusy(() -> assertThat(isTrackerActive(), is(false)));
+        assertBusy(
+            () -> {
+                var res = executeOnSubscriber(
+                    "SELECT s.subname, sr.srrelid::text, sr.srsubstate, sr.srsubstate_reason" +
+                        " FROM pg_subscription s" +
+                        " LEFT JOIN pg_subscription_rel sr ON s.oid = sr.srsubid" +
+                        " ORDER BY s.subname");
+                assertThat(printedTable(res.rows()), CoreMatchers.is("sub1| doc.t1| e| Tracking of metadata failed for subscription 'sub1' with unrecoverable error, stop tracking.\n" +
+                    "Reason: Publication 'pub1' unknown\n"));
+            }, 20, TimeUnit.SECONDS
+        );
+
+        response = executeOnSubscriber("INSERT INTO t1 (id) VALUES(3)");
```

Comment:
this assertion currently fails, i.e tracking is stopped but table is still "locked".

Reply:
I think this is actually okay. Users will have to take a look at what happened by checking the subscription tables/logs, and if they're okay with the publication being gone they can drop the subscription and after that it should work(?)

Reply:
thx for the feedback! I will check with drop sub - if it works I will update documents - we already say that replication stops but will add a note that table is read-only and one have to use DROP SUB

Reply:
> probably need this test case?

yep I think it is good to keep it